pages/resources/message.mdx

```diff
@@ -1301,3 +1304,210 @@ Returns embed data from a list of URLs.
 | Field  | Type                                 | Description                   |
 | ------ | ------------------------------------ | ----------------------------- |
 | embeds | array[[embed](#embed-object) object] | The found embeds for the URLs |
+
+### Poll Object
```

Comment:
This shouldn't be under the endpoints header, it should be above it, with only the actual endpoints added here